src/ucp/core/ucp_worker.c

```diff
@@ -404,6 +406,8 @@ ucp_worker_add_iface(ucp_worker_h worker, ucp_rsc_index_t tl_id,
         if (status != UCS_OK) {
             goto out_close_iface;
         }
+
+        worker->arm_ifaces_mask |= UCS_BIT(tl_id);
```

Comment:
you are right, i don't
will change this to a list instead of bitmap to support larger values of tl_id